transaction/core/tests/input_rules.rs

```diff
@@ -13,10 +13,9 @@ use mc_transaction_core::{
 use mc_util_from_random::FromRandom;
 use mc_util_test_helper::{RngType, SeedableRng};
 
-use util::{
-    create_test_tx, create_test_tx_with_amount_and_comparer_and_recipients,
-    INITIALIZE_LEDGER_AMOUNT,
-};
+use util::{create_test_tx, create_test_tx_with_amount_and_comparer_and_recipients};
+
+use mc_ledger_db::test_utils::INITIALIZE_LEDGER_AMOUNT;
```

Comment:
I moved this this `use` from `transaction/core/tests/util/mod.rs`. This seems OK to me but would love a doublecheck this that this is what we intend.